docs/devel/api-change-guidelines.md

```diff
@@ -0,0 +1,21 @@
+# API Change Guidelines
+
+Freely change pkg/api/types.go. 
+
+Update the most recent versioned package's conversion functions. (In, e.g., pkg/api/v1beta1/conversion.go)
+
+If necessary, you may add fields to the versioned package's objects. You may not remove fields or rename them (or change the json/yaml tags), because that will break clients.
```

Comment:
Also:
All newly added fields must be documented using field description tags. We have a pre-submit hook to verify that descriptions exist for all API fields.